### PR TITLE
Provide Docker run profile for templates

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

Relates to https://github.com/arcus-azure/arcus.templates/issues/108

### CODE REVIEW

File: src/Arcus.Templates.ServiceBus.Queue/Arcus.Templates.ServiceBus.Queue.csproj [hunk 1]
```diff
@@ -37,14 +37,12 @@
   <PropertyGroup>
     <DefineConstants>$(DefineConstants);</DefineConstants>
     <ExcludeSerilog>false</ExcludeSerilog>
+    <UserSecretsId>d7596530-df47-4168-8585-a0cbbb74f2a9</UserSecretsId>
```
Comment (stijnmoreels) [inline src/Arcus.Templates.ServiceBus.Queue/Arcus.Templates.ServiceBus.Queue.csproj]: Is this something that has to change each new project? 😦

**Comment by tomkerkhove** [comment]
It works:
image

**Comment by tomkerkhove** [comment]
Aaah test failures, will dig into them later today

**Comment by stijnmoreels** [comment]
> Aaah test failures, will dig into them later today

Already done for you by merging with `master`.

**Pull request merged**